parlai/core/params.py

```diff
@@ -772,6 +772,16 @@ def add_distributed_training_args(self):
         grp.add_argument(
             '--distributed-world-size', type=int, help='Number of workers.'
         )
+        grp.add_argument(
+            '--ddp-backend',
+            choices=['ddp', 'zero2', 'zero3'],
```

Comment:
Hmm should we even give `'zero3'` as an option for the time being? (Don't really care either way)